staging.gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json

```diff
@@ -290,7 +290,7 @@
     "environment": "stagingdatastage",
     "hostname": "staging.gen3.biodatacatalyst.nhlbi.nih.gov",
     "revproxy_arn": "arn:aws:acm:us-east-1:895962626746:certificate/1876fa51-031e-4aa9-ab95-7da182a52c7f",
-    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/gtexdictionary/3.6.2/schema.json",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/gtexdictionary/4.0.1/schema.json",
```

Comment:
this is a big jump, I'm worried there may be breaking changes? We may want to verify with bios/user services this isn't going to break things

Reply:
This is promoting changes from preprod so this will only cause problems if the data that has been previously submitted to staging is not compliant with this new DD version 🤷‍♂️ 
cc: @qiongl-CTDS @svburke ?